nservicebus/sqlserver/multiple-databases.md

```diff
@@ -1,21 +1,66 @@
 ---
-title: Multi-database support
-summary: How to configure SQL Server transport to use multiple instances of the database and route messages between them.
+title: SQL Server transport setup - default, multi-schema, multi-catalog and multi-instance
+summary: How to configure and manage SQL Server transport in default, multi-schema, multi-catalog and multi-instance modes
 tags:
 - SQL Server
 - Transports
 ---
 
-The SQL Server transport supports selecting, on per-endpoint basis, where the table queues should be created. The selection can be done on multiple levels:
+In the default SQL Server transport setup, the messages and other data are stored in multiple tables in a single database. The default schema is `dbo`, but it is possible to [specify a custom schema](/nservicebus/sqlserver/configuration.md#custom-database-schemas).
 
- * different schemas in a single database
- * different databases in a single SQL Server instance
- * different SQL Server instances
+In some environments it might be necessary to use a more advanced setup. The supported options are:
+ * ***default***: single schemas in a single database (Versions 1.x and higher)
+ * ***multi-schema***: multiple schemas in a single database (Versions 2.1 and higher)
+ * ***multi-catalog***: multiple databases in a single SQL Server instance (Versions 2.1 and higher)
+ * ***multi-instance***: multiple databases in various SQL Server instances (Versions 2.1 to 2.x).
 
-The transport will route messages to destination endpoints based on the configuration. If no specific configuration has been provided for a particular destination endpoint, the transport assumes the destination has the same configuration (schema, database and instance name/address) as the sending endpoint. If this assumption turns out to be false (the transport cannot connect to destination queue), an exception is thrown immediately. There is no store-and-forward mechanism on the transport level (and hence -- no dead-letter queue).
+The transport will route messages to destination endpoints based on the configuration. If no specific configuration has been provided for a particular destination endpoint, the transport assumes the destination has the same configuration (schema, database and instance name) as the sending endpoint. If the destination endpoint has a different configuration and it hasn't been provided, then exception will be thrown immediately (since the transport cannot connect to the destination queue).
```

Comment:
Do we want to say that this not apply to ReplyTo or Publish in which case non-default configuration will be taken from header or subscription store?